openerp/addons/base/i18n/base.pot

```diff
@@ -92,6 +112,14 @@ msgid "\n"
 msgstr ""
 
 #. module: base
+#: code:addons/model.py:146
```

Comment:
We have that already for [other terms](https://github.com/odoo/odoo/blob/d780f9476d43eb4fd25c852dd4e691002f4a6dcf/openerp/addons/base/i18n/base.pot#L14350) so I don't think it's related to that PR but yes, wrong path